demo-app/src/app/pagination/pagination.component.html

```diff
@@ -0,0 +1,194 @@
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Listen to page change</h5>
+
+  <div class="row m-valign-wrapper">
+    <mz-pagination class="col s12 m6"
+      (pageChange)="pageChange($event)"
+      [itemsPerPage]="10"
+      [totalItems]="100"
+    >
+    </mz-pagination>
+
+    <p class="col s12 m6">
+      You can use the ouput property <code class="language-markup">changePage</code> to call a function when page changed.
+    </p>
+</div>
+
+<div class="section">
+  <h5 class="light">Playground</h5>
+
+  <p>
+    The input properties of the <code class="language-markup">mz-pagination</code> directive makes the form-control completely dynamic. Change the configuration below and see how the checkbox reacts.
+  </p>
+
+  <div class="row">
+
+    <div class="col s12 m6">
+      <mz-pagination
+        [currentPage]="paginationCurrentPage"
+        [enableFirstAndLastPageButtons]="paginationEnableFirstAndLastPageButtons"
+        [itemsPerPage]="paginationItemPerPage"
+        [maxPageButtons]="paginationMaxPageButtons"
+        [totalItems]="paginationTotalItems"
+        (pageChange)="playgroundPageChange($event)"
+      >
+      </mz-pagination>
+    </div>
+
+    <div class="col s12 m6">
+
+      <mz-select-container>
+        <select mz-select
+          id="pagination-current-page"
+          [label]="'Current page'"
+          [(ngModel)]="paginationCurrentPage"
+        >
+          <option *ngFor="let pageOption of pageOptions" [ngValue]="pageOption">{{ pageOption }}</option>
+        </select>
+      </mz-select-container>
+
+      <mz-input-container>
+        <input mz-input
+          id="pagination-item-per-page"
+          [label]="'Item per page'"
+          [(ngModel)]="paginationItemPerPage"
+          type="number">
+      </mz-input-container>
+
+      <mz-input-container>
+        <input mz-input
+          id="pagination-max-page-buttons"
+          [label]="'Max page button'"
+          [(ngModel)]="paginationMaxPageButtons"
+          type="number">
+      </mz-input-container>
+
+      <mz-input-container>
+        <input mz-input
+          id="pagination-total-items"
+          [label]="'Total items'"
+          [(ngModel)]="paginationTotalItems"
+          type="number">
+      </mz-input-container>
+
+      <mz-checkbox-container>
+        <input mz-checkbox
+          id="checkbox-enable-first-last-page-buttons"
+          [label]="'Enabled first and last page buttons'"
+          [filledIn]="true"
+          [(ngModel)]="paginationEnableFirstAndLastPageButtons"
+          type="checkbox">
+      </mz-checkbox-container>
+    </div>
+  </div>
+</div>
+
+<div class="section">
+
+  <h5 class="light">HTML Structure</h5>
+    <app-code-snippet>
+&lt;mz-pagination
+  (pageChanged)="onPageChange($event)"
+  [currentPage]="1"
+  [enableFirstAndLastButton]="false"
```

Comment:
here you wrote `enableFirstAndLastButton` with no "s" at the end and still does not fit the `@Input enableFirstAndLastPageButtons` property in the component